v20.2/performance-benchmarking-with-tpc-c-100k-warehouses.md

```diff
@@ -11,16 +11,22 @@ This page shows you how to reproduce [CockroachDB's TPC-C performance benchmarki
 <div class="filters filters-big clearfix">
   <a href="performance-benchmarking-with-tpc-c-10-warehouses.html"><button class="filter-button">10</button></a>
   <a href="performance-benchmarking-with-tpc-c-1k-warehouses.html"><button class="filter-button">1000</button></a>
-  <a href="performance-benchmarking-with-tpc-c-10k-warehouses.html"><button class="filter-button">10,000</button></a>
-  <button class="filter-button current"><strong>100,000</strong></button>
+  <a href="performance-benchmarking-with-tpc-c-10k-warehouses.html"><button class="filter-button">13,000</button></a>
+  <button class="filter-button current"><strong>140,000</strong></button>
```

Comment:
Note that I am also planning to update the page URLs so the page text does not say "140" but the URL says "100".

In fact I think it's a bad idea to hardcode the run sizes into the URLs.  What do y'all think of something like 'S, M, L, XL' - open to better ideas